requirements-iree-pinned.txt

```diff
@@ -2,6 +2,6 @@
 
 # Keep these versions synced with SHORTFIN_IREE_GIT_TAG in shortfin/CMakeLists.txt
```

Comment:
You can also try to update https://github.com/nod-ai/shark-ai/blob/13781bb2f01f5aa96c9f0e46cc127568b161510c/shortfin/CMakeLists.txt#L46-L50, or let it drift a bit

Reply:
If you specifically want IREE runtime changes in shortfin, then you do need to update that, not just the python package requirements.